plugins/producer_plugin/include/eosio/producer_plugin/subjective_billing.hpp

```diff
@@ -119,50 +119,50 @@ class subjective_billing {
    void disable_account( chain::account_name a ) { _disabled_accounts.emplace( a ); }
 
    /// @param in_pending_block pass true if pt's bill time is accounted for in the pending block
-   void subjective_bill( const transaction_id_type& id, const fc::time_point& expire, const account_name& first_auth,
+   void subjective_bill( const transaction_id_type& id, const fc::time_point& expire, const account_name& authorizer,
```

Comment:
here again change authorizer to resource_payer and in other places in the code

Reply:
Done